assets/less/components/donation-elements.less

```
@@ -1,30 +1,49 @@
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
 .modal {
-    width:50%;
-    margin-left:auto;
-    margin-right:auto;
-    padding: 5px;
-    border: thin #000 solid;
-    background-color:#fff;
-    z-index:3; /* places the modal dialog on top of everything else */
-    position:fixed;
-    top:25%;
-    left:25%;
-    display:none;
-}
+    top: calc(50% - 250px);
+
+    #close-modal {
+        cursor: pointer;
+
+        svg {
+            stroke: @colors[grey];
+        }
+        &:hover svg {
+            stroke: @colors[black];
+        }
+    }
+
+    form {
+        .amount-selection {
+            label {
+                &:extend(.h-10, .pt-1, .pl-3, .pb-1, .pr-3, .border, .border-solid, .border-grey, .bg-white, .font-xl, .font-bold);
+                cursor: pointer;
+                box-sizing: border-box;
+                &:hover {
+                    &:extend(.border-blue);
+                }
+                &:last-child {
+                    &:extend(.pt-0, .pb-0, .pr-0, .bg-grey-lighter);
+                    flex-basis: @widths[w-40];
+                    #amount-other {
+                        &:extend(.w-full, .h-full, .pl-2, .border-0, .border-l, .border-solid, .border-grey, .font-xl);
+                        box-sizing: border-box;
+                    }
```

Let's style the placeholder with a smaller font size so it looks nicer.

```suggestion
                    }
                    
                    &::-moz-placeholder,
                    &::-webkit-input-placeholder,
                    &::placeholder {
                        &:extend(.font-md);
                    }
```